plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1951,6 +1953,27 @@ var _ = Describe("validator", func() {
 
 				Expect(err).To(HaveOccurred())
 			})
+
+			It("should fail to change Seed name, because cloud provider for new Seed is not equal to cloud provider for old Seed", func() {
+				oldSeedName := fmt.Sprintf("new(%s)", seedName)
```

Comment:
```suggestion
				oldSeedName := fmt.Sprintf("old-%s", seedName)
```

Reply:
ops forgot to change this after I modified the variable name.